client/src/components/Navigation.jsx

```diff
@@ -33,16 +39,22 @@ const Navigation = () => {
         <nav className="bg-secondary-1 ma0 pa2">
             <ul className="flex justify-between items-center list">
                 <aside className="f3">
-                    <Link className="link primary-2 hover-primary-1" to="/">
+                    <Link
+                        className="fink ts-1 link primary-2 hover-primary-1"
+                        to="/"
+                    >
                         AniArch
                     </Link>
                 </aside>
 
                 <main className="flex">
                     {navRoutes.map(
                         (route, i) =>
-                            !route.loggedIn && (
-                                <li key={"nav-link-" + i} className="ml4">
+                            !route.loggedOut && (
```

Comment:
For the navigation array, maybe it can have one property named `loggedIn` (rather than `loggedIn` or `loggedOut`, and when mapping on the `navRoutes`, a filter can be applied to find which routes should be rendered.

```
navRoutes.filter((route) => !route.loggedIn).map(...) // should only render loggedOut routes
```

or

```
navRoutes.filter((route) => route.loggedIn).map(...) // should only render loggedIn routes
```

Reply:
Yeah `loggedIn` should be a boolean, and I'm more in the should logged in stuff if logged in and not by default, so the second option of what Corrina said would be great

Reply:
Whoops! Yeah, I initiailly had it like this because I wanted to have Home link to `/` in both cases, but i changed that in this PR to use unique routes. Will update this evening!

Reply:
FIXED!